Route refresh: added `incidents` (#5749)
mapbox/mapbox-navigation-android: CHANGELOG.md

- Added `TollCollection#name` field which contains a name of the toll collection point, when available. [#5784](https://github.com/mapbox/mapbox-navigation-android/pull/5784)
- Added `ReplayRouteOptions#frequency` to allow adjusting the simulated location frequency. [#5724](https://github.com/mapbox/mapbox-navigation-android/pull/5724)
- Added `MapboxReplayer#eventRealtimeOffset` to allow adjusting simulated locations timestamps for more accurate replays. [#5748](https://github.com/mapbox/mapbox-navigation-android/pull/5748)

> - Added update of `incidents` for route refresh. For now `incidents` are refreshed along a route. [#5749](https://github.com/mapbox/mapbox-navigation-android/pull/5749)

Reviewer: Please also add an instrumentation test for this in `com.mapbox.navigation.instrumentation_tests.core.RouteRefreshTest`.

> - Added update of `incidents` for route refresh. For now `incidents` are refreshed along a route. [#5749](https://github.com/mapbox/mapbox-navigation-android/pull/5749)

Author: We usually start changelog lines from Added/Updated/Removed. I'd change to align 
```suggestion
- Updated feature `route refresh`: now also supports refreshing `incidents` along the route found in `RouteLeg#incidents` and `RoadObjectsOnRouteObserver`. [#5749](https://github.com/mapbox/mapbox-navigation-android/pull/5749)
```
